### ISSUE TITLE
Is there possibility to add verbose=True, to see increase/decrease in lr as training progresses.

### ISSUE THREAD

**Issue by rkakash59** [katsura-jp/pytorch-cosine-annealing-with-warmup#7]

**Comment by katsura-jp** [comment]
Hi, @rkakash59 ! thank you comment.
I hadn't noticed the `verbose` argument before.
I'm trying to make the implementation as similar to PyTorch's lr_scheduler as possible, so I'll add it.
